app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -986,9 +986,9 @@ $include-html-global-classes: $include-html-classes;
 // $include-html-panel-classes: $include-html-classes;
 
 // We use these to control the background and border styles
-// $panel-bg: scale-color($white, $lightness: -5%);
-// $panel-border-style: solid;
-// $panel-border-size: 1px;
+ $panel-bg: #FFFFFF;
```

Comment:
Color `#FFFFFF` should be written as `#ffffff`<br>Line should be indented 0 spaces, but was indented 1 spaces